tournaments/index.js

```diff
@@ -879,7 +879,13 @@ CommandParser.commands.tournament = function (paramString, room, user) {
 		}
 
 		var tour = createTournament(room, params.shift(), params.shift(), params.shift(), Config.istournamentsrated, params, this);
-		if (tour) this.privateModCommand("(" + user.name + " created a tournament in " + tour.format + " format.)");
+		if (tour) {
+			this.privateModCommand("(" + user.name + " created a tournament in " + tour.format + " format.)");
+			if (Config.tourannouncements && Config.tourannouncements[room.id]) {
+				var tourRoom = Rooms.search(Config.tourroom || 'tournaments');
+				if (tourRoom) tourRoom.addRaw('<div class="infobox">' + Tools.getFormat(tour.format).name + ' tournament created in the <u><a href="/' + room.id + '" class="ilink">' + room.title + '</a></u> room.</div>');
```

Comment:
Generally, for UI messages, more concise is better. Users will be reading it hundreds of times, after all.